src/main/webapp/app/iris/http-message.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { IrisClientMessage, IrisMessage } from 'app/entities/iris/iris.model';
+
+type EntityResponseType = HttpResponse<IrisMessage>;
+type EntityArrayResponseType = HttpResponse<IrisMessage[]>;
+
+/**
+ * Provides a singleton instance of http client to perform CRUD operations.
+ */
+@Injectable({ providedIn: 'root' })
+export class IrisHttpMessageService {
+    public resourceUrl = 'api/iris/sessions';
+
+    constructor(private httpClient: HttpClient) {}
+
+    /**
+     * creates a message for a session
+     * @param {number} sessionId
+     * @param {IrisClientMessage} message
```

Comment:
```suggestion
     * @param {IrisMessage} message
```
could you change this too?

Reply:
This method is only indented to send client messages, I adjusted the signature